<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 7);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="profile_cap_LBL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="profile_TW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>75</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>27</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>T [Â°C]</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>Gennaio</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Febbraio</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>Marzo</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>Aprile</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>Maggio</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>Giugno</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>Luglio</string>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>Agosto</string>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>Settembre</string>
    </property>
   </item>
   <item row="9" column="0">
    <property name="text">
     <string>Ottobre</string>
    </property>
   </item>
   <item row="10" column="0">
    <property name="text">
     <string>Novembre</string>
    </property>
   </item>
   <item row="11" column="0">
    <property name="text">
     <string>Dicembre</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="import_BTN">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Importa Dati</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_BTN">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Carica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_BTN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Nuovo</string>
   </property>
  </widget>
  <widget class="Line" name="elem_name_LN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>580</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_BTN">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Conferma e ESCI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_BTN">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_BTN">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>ESCI</string>
   </property>
  </widget>
  <widget class="QLabel" name="profile_LBL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>5</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profile Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="month_BTN">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Settembre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_BTN">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>470</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Reset valori</string>
   </property>
  </widget>
  <widget class="Line" name="bottom_LN">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>500</y>
     <width>600</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="year_BTN">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Anno  &lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="day_BTN">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Giorno: 30</string>
   </property>
  </widget>
  <widget class="Line" name="elem_name_LN_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>580</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="top_LN">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>600</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="plt_WGT" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>411</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="step_BTN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Uniform for all days</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_BTN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);</string>
   </property>
   <property name="text">
    <string>Creazione guidata</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
